taskcluster/ac_taskgraph/transforms/complete.py

```diff
@@ -31,67 +26,63 @@ def resolve_keys(config, tasks):
     for task in tasks:
         for key in ("notifications",):
             resolve_keyed_by(
-                task, key,
+                task,
+                key,
                 item_name=task["name"],
                 **{
                     "geckoview-bump": _get_geckoview_bump(config, task),
-                }
+                },
             )
         yield task
 
 
-_GECKOVIEW_NIGHTLY_BUMP_BRANCH_PATTERN = re.compile(r"^GV-nightly-(?P<date>\d{8})-(?P<time>\d{6})$")
-
 def _get_geckoview_bump(config, task):
-    if all((
-        task["name"] == "pr",  # Filter out pr-1 and pr-2
-        config.params["owner"] == "MickeyMoz@users.noreply.github.com",
-        config.params["head_repository"] == "https://github.com/MickeyMoz/android-components",
-        config.params["tasks_for"] == "github-pull-request",
-        _GECKOVIEW_NIGHTLY_BUMP_BRANCH_PATTERN.search(config.params["head_ref"]),
-    )):
+    if all(
+        (
+            task["name"] == "pr",  # Filter out pr-1 and pr-2
+            config.params["owner"] == "github-actions[bot]@users.noreply.github.com",
+            config.params["head_repository"] == config.params["base_repository"],
```

Comment:
Note to future readers: Github Action creates branches on the base repo, not on a fork anymore.